RELEASING.md

```diff
@@ -50,15 +50,18 @@ If you're running in linux and would like to use the GPG agent to remember your
 you can configure the following in `<your-home-directory>/.gradle/gradle.properties`:
 
     ```text
+    ossrhUsername=<generated-token-user>
+    ossrhPassword=<generated-token-key>
+    signingUseGpgCmd=true
+
```

Comment:
Nit: This whitespace seems unnecessary (and if you do want to split them into two groups, it seems like `signingUseGpgCmd` belongs in the second?)